test/basic.js

```diff
@@ -549,6 +577,8 @@ util.testSinglePlatform('ignore test: string in array', createIgnoreTest, ['igno
 util.testSinglePlatform('ignore test: string', createIgnoreTest, 'ignorethis', 'ignorethis.txt')
 util.testSinglePlatform('ignore test: RegExp', createIgnoreTest, /ignorethis/, 'ignorethis.txt')
 util.testSinglePlatform('ignore test: Function', createIgnoreTest, function (file) { return file.match(/ignorethis/) }, 'ignorethis.txt')
+util.testSinglePlatform('ignore test: Function', createIgnoreFromGlobalFileTest, 'ignorethis.txt')
```

Comment:
Also you should use another filename for this test, otherwise I believe it will create a false positive for the other ignore tests.